dashboard/src/utils/axiosInstance.js

```diff
@@ -17,7 +17,7 @@ const axiosInstance = axios.create({
 });
 
 axiosInstance.interceptors.request.use(async (req) => {
-  token = Cookies.get("token") ? Cookies.get("token") : null;
+  token = Cookies.get("token");
   req.headers.Authorization = `Bearer ${token}`;
```

Comment:
If `token` is false-y (e.g., `undefined`, `null`, or `""`), shouldn't we be omitting the request `Authorization` header?  (Here and at line 15?)

Reply:
Added the check